src/nvim/getchar.c

```diff
@@ -263,18 +265,19 @@ add_buff (
         STRLEN(buf->bh_first.b_next->b_str + buf->bh_index) + 1);
   buf->bh_index = 0;
 
-  ssize_t len;
-  if (buf->bh_space >= (int)slen) {
+  size_t len;
```

Comment:
Just checked. It's OK because at this point `slen` won't be `< 0`. I checked git blame and I was the one who added `ssize_t` here :)